as_new_user.md

```diff
@@ -0,0 +1,20 @@
+# Create your account on SRC
+
+This manual shows you how to get access to the (Jupyter Hub) workspace/environment on the Surf research cloud.
+
+After creating your own account, you will:
+- Get your own account (i.e., your own Jupyter Lab environment).
+- Have access to our shared memory (e.g. 1.5TB) where the forcing data is available in `/data/volume_2/`. 
+- All changes in your local home Jupyter lab environment will be stored (if you save them) after you log-out.
```

Comment:
Local might be confusing here. You probably mean the users home directory on the SURF Research cloud virtual machine. The saved files are not automatically saved to you local laptop.
```suggestion
- All changes in your home directory on the Jupyter lab workspace will be stored (if you save them) after you log-out.
```

Reply:
Yes, exactly. Thanks for the correction. Here I take it for granted that the user would realize everything will be on the path on SRC. It is good to clarify this.